pkg/secrets/secrets.go

```diff
@@ -61,6 +61,7 @@ func GenerateUserData(platformType oconfig.PlatformType, publicKey ssh.PublicKey
 
 // generateUserDataWithPubKey returns the Windows user data for the given pubKey
 func generateUserDataWithPubKey(pubKey string) string {
+	windowsExporterPort := "9182"
```

Comment:
Can we use `metrics.Port` here instead?

Reply:
we cannot as it causes a circular dependency

Reply:
`metrics` pkg imports `nodeconfig` imports `secrets`
so we can't import `metrics` from `secrets`

points to a larger issue with our pkg structuring especially where global vars/constants are defined-- e.g. I don't think "helper" packages like `metrics` should import more core packages like `nodeconfig`